jitpack.yml

```diff
@@ -0,0 +1,6 @@
+jdk:
+  - openjdk11
+before_install:
+  - ./prepareEnvironment.sh
+install:
+  - mvn clean install -DskipTests
```

Comment:
Thanks! Why skipping the tests, wouldn't make sense to run them to make sure the "released" artifact is sound?